app/templates/settings.hbs

```diff
@@ -1,139 +1,153 @@
-      </ul>
-    </section>
-  {{/if}}
-
-  {{#if showAllowConfigImportsSwitch}}
-    <section class="settings-section config-import">
-      <h2 class="small-title display-inline">Config Import</h2>
-
-      <ul class="settings-list--columns">
-        <li>{{settings-switch active=model.settings.allow_config_imports repo=repo description="Allow importing config files from this repository" key="allow_config_imports"}}</li>
-      </ul>
-    </section>
-  {{/if}}
-
-  <section class="settings-section">
-    <h2 class="small-title">Environment Variables</h2>
-
-    <p>Notice that the values are not escaped when your builds are executed. Special characters (for bash) should be escaped accordingly.</p>
-
-    <ul class="settings-list--envvars">
-      {{#each envVars as |envVar|}}
-        {{env-var envVar=envVar}}
-      {{/each}}
-    </ul>
-
-    <div id='env-var-secrecy'>
-      <p>
-        {{svg-jar 'icon-megaphone' class='icon icon-megaphone'}}
-        <span class="label-align">
-          Please make sure your secret key is never related to the repository, branch name, or any other guessable string.
-          For more tips on generating keys
-          <a href='https://docs.travis-ci.com/user/best-practices-security/#Steps-Travis-CI-takes-to-secure-your-data'>
-          read our documentation</a>.
-        </span>
-      </p>
-    </div>
-
-    {{add-env-var repo=repo}}
-
-  </section>
-
-  <section class="settings-section">
-      <h2 class="small-title display-inline">Cron Jobs</h2>
-
-    <ul class="settings-list--crons">
-      {{#each cronJobs as |cron|}}
-        {{cron-job cron=cron}}
-      {{/each}}
-    </ul>
-
-    {{#if sortedBranchesWithoutCron.length}}
-        {{add-cron-job branches=sortedBranchesWithoutCron}}
-    {{else}}
-        <p class="text-medium">There are no branches you can currently add crons to.</p>
-    {{/if}}
-
-  </section>
-
-{{#if (and config.endpoints.sshKey repo.private)}}
-  <section class="settings-section" data-test-ssh-key-section>
-    <h2 class="small-title">SSH Key</h2>
-
-    {{#if model.customSshKey}}
-      {{ssh-key key=model.customSshKey sshKeyDeleted="sshKeyDeleted" pushAccess=model.hasPushAccess}}
-    {{else}}
-      {{ssh-key key=model.sshKey}}
-
-      {{#if model.hasPushAccess}}
-        {{add-ssh-key repo=repo sshKeyAdded="sshKeyAdded"}}
-      {{/if}}
-    {{/if}}
-
-    {{#unless model.hasPushAccess}}
-      <p>You don't have sufficient permissons to add or remove ssh keys on this repository.</p>
-    {{/unless}}
-
-  </section>
-{{/if}}
+	<section class="settings-section">
+		<h2 class="small-title">General</h2>
+		<ul class="settings-list--columns">
+			<li>
+				{{settings-switch active=model.settings.build_pushes repo=repo description="Build pushed branches" key="build_pushes"}}
+				<a href="https://docs.travis-ci.com/user/web-ui/#build-pushed-branches" title="about branch updates"
+					class="settings-tooltip">
```

Comment:
For consistency, please either put all attributes in one line or put each one on its own line (when it's too long), like this:
```
<a href="https://docs.travis-ci.com/user/web-ui/#build-pushed-branches" 
  title="about branch updates"
	class="settings-tooltip"
>
```
When multi-line, please put closing bracket to the new line too.